packages/eslint-plugin-react-hooks/src/RulesOfHooks.js

```diff
@@ -169,17 +139,6 @@ export default {
         : node => {
             return context.sourceCode.getText(node);
           };
-    /**
-     * SourceCode#getScope that also works down to ESLint 3.0.0
-     */
-    const getScope =
```

Comment:
I originally left this in, since none of what I removed was using it (and it's meant to act as a compatibility layer for any future use), but ESLint was failing the build due to it not being referenced. Happy to add it back with a disable, if you think that would be better.